aip/general/0203.md

```diff
@@ -42,6 +42,14 @@ backwards-compatility. Nontheless, this annotation **must not** be omitted.
 only, and does not itself add any validation. The purpose is to consistently
 document this behavior for clients.
 
+### Annotating Name
+
+The `name` field of a resource **must not** be annotated.
```

Comment:
Should we also adjust Immutable Potential use case section? At the moment [it contains](https://github.com/aip-dev/google.aip.dev/blob/master/aip/general/0203.md?plain=1#L144-L146)
```
Potential use cases for immutable fields (this is not an exhaustive list) are:
- Names or IDs which are set on creation and then used as a primary key.
```